testing/sdk_utils.py

```diff
@@ -43,7 +43,7 @@ def get_zk_path(service_name):
 
 @functools.lru_cache()
 def dcos_version_less_than(version):
-    return shakedown.dcos_version_less_than("1.10")
+    return shakedown.dcos_version_less_than(version)
```

Comment:
I hadn't been following that thread. So we've had stable builds for a while now because I made a typo when I created `dcos_version_less_than`? Ha!

Reply:
I think @benclarkwood has a fix pending (I haven't had a chance to look at it again). Just wanted to give you a heads-up in case you see hello-world-19 tests failing.

Reply:
Appreciated! I was hitting the `run` button in TeamCity to see if hello-world-19 was a flake. (It worked for me locally so I assumed so.)